qlib/SoapClient.qm

```diff
@@ -360,10 +360,12 @@ public namespace SoapClient {
             @throw WSDL-BINDING-ERROR the binding is not assigned to a SOAP operation in the WSDL
             @throw HTTP-CLIENT-RECEIVE-ERROR this exception is thrown when the SOAP server returns an HTTP error code; if a SOAP fault is returned, then it is deserialized and returned in the \a arg key of the exception hash
 
-            @note this method can throw any exception that @ref Qore::HTTPClient::send() "HTTPClient::send()" can throw as well as any XML parsing errors thrown by @ref Qore::XML::parse_xml() "parse_xml()"
+            @note
+            - this method can throw any exception that @ref Qore::HTTPClient::send() "HTTPClient::send()" can throw as well as any XML parsing errors thrown by @ref Qore::XML::parse_xml() "parse_xml()"
+            - it's recommended to use callOperation() instead, which was a cleaner and more extensible API
         */
-        any call(string operation, any args, *hash header, *hash nsh, *string binding) {
-            return makeCallIntern(NOTHING, operation, args, ("soap_header": header, "http_header": nsh, "binding": binding));
+        auto call(string operation, auto args, *hash<auto> header, *hash<auto> nsh, *string binding) {
```

Comment:
can you maybe rename the `header` and `nsh` to the `soap_header` and `http_header` all over the module?

Reply:
ok will do

Reply:
done